query-engine/connector-test-kit-rs/query-tests-setup/src/logging.rs

```diff
@@ -66,19 +68,38 @@ fn create_env_filter(log_queries: bool, qe_log_level: &str) -> EnvFilter {
 /// specific test outputs for readability.
 ///
 /// It is used from test_macros.
-pub struct PrintWriter;
+pub struct PrintWriter {
+    tx: LogEmit,
+}
+
+impl PrintWriter {
+    fn new(tx: LogEmit) -> Self {
+        Self { tx }
+    }
+}
 
 impl tracing_subscriber::fmt::MakeWriter<'_> for PrintWriter {
     type Writer = PrintWriter;
 
     fn make_writer(&self) -> Self::Writer {
-        PrintWriter
+        PrintWriter::new(self.tx.clone())
     }
 }
 
 impl std::io::Write for PrintWriter {
     fn write(&mut self, buf: &[u8]) -> Result<usize, std::io::Error> {
-        eprint!("{}", std::str::from_utf8(buf).unwrap_or("<invalid UTF-8>"));
+        let log = std::str::from_utf8(buf).unwrap_or("<invalid UTF-8>");
+
+        if log.contains("quaint") {
+            let remove = ["[3m", "[0m", "[1m", "\u{1b}", "[34m", "[2m", "[32m", "[35m", "\n"];
```

Comment:
Have you thought about using https://docs.rs/strip-ansi-escapes/latest/strip_ansi_escapes instead ? Might be a bit overkill but it feels like we're gonna have to manage this list overtime if we ever change colors